src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,37 @@
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+interface Props {
+  people: Person[];
+  personSlug: string | undefined;
```

Comment:
handle it in the upper component
```suggestion
  personSlug: string;
```